components/bsx/Gallery/Item/AvailableActions.vue

```diff
@@ -98,6 +109,28 @@ export default class AvailableActions extends mixins(
     return actionComponent[this.selectedAction]
   }
 
+  public async created(): Promise<void> {
+    setTimeout(() => {
+      const { api } = Connector.getInstance()
+      this.minimumOfferAmount = parseFloat(
+        formatBalance(
+          api?.consts?.marketplace?.minimumOfferAmount?.toString(),
+          12,
+          false
+        ).replace(/,/g, '')
+      )
+
+      this.isMakeOffersDisabled =
+        !this.isMakeOffersAllowed || this.minimumOfferAmount > this.balance
+
+      if (!this.isMakeOffersAllowed) {
+        this.tooltipOfferLabel = this.$t('tooltip.makeOfferDisabled')
+      } else if (this.minimumOfferAmount > this.balance) {
+        this.tooltipOfferLabel = this.$t('tooltip.makeOfferNotEnoughBalance')
+      }
+    }, 3000)
+  }
+
```

Comment:
May I ask Why are you using set timeout ?

Reply:
There is onApiConnected hook

Reply:
oh didnt know about the hook, will use that thanks